target/linux/ipq806x/patches-4.19/0049-PM-OPP-Support-adjusting-OPP-voltages-at-runtime.patch

```diff
@@ -1,7 +1,14 @@
-From c949f08cf20fe82971fbdb4015daa38210da492e Mon Sep 17 00:00:00 2001
```

Comment:
Commit subject/description should be improved.

Reply:
Is it good now? I put the merged patch

Reply:
I've meant this one ![image](https://user-images.githubusercontent.com/43848/71447899-06429c00-2735-11ea-9760-534efc2d5909.png) as it's not possible to comment on this...

Reply:
Perhaps something like this?

`ipq806x: opp/core: update patch allowing adjusting of OPP voltages at runtime`